tests/units/utilities/test_distribution.py

```diff
@@ -16,9 +16,20 @@
     LocalDaskDistributor,
     MultiprocessingDistributor,
 )
+from tsfresh.utilities.profiling import get_n_jobs, set_n_jobs
 
 
 class MultiprocessingDistributorTestCase(TestCase):
+    def test_n_jobs(self):
```

Comment:
Could you make sure that the default setting (as it was before running this test) is reset after the test? (also if the test failed).
Just to make sure that in case any other test assumes the default value is still set.

Reply:
Yup, I can do that. Should I make the changes and open as a new PR?

Reply:
You can just add the changes to this PR as they belong together :)